test/optim/test_optimize.py

```diff
@@ -54,6 +58,10 @@ def forward(self, X):
 
 class SquaredAcquisitionFunction(AcquisitionFunction):
     def __init__(self, model=None):
+        """
+        Args:
+            model: A fitted model. Defaults to None.
```

Comment:
I think we don't need this docstring - this is just a dummy here used for testing and shouldn't be used by the end user. Let's just `noqa` this away?